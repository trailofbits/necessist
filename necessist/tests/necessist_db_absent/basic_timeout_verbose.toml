args = ["--no-sqlite", "--root=fixtures/basic", "--timeout=5", "--verbose"]
stderr = ""

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 0
