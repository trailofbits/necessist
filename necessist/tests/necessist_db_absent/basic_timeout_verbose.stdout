4 candidates in 1 test file
examples/basic/src/lib.rs: dry running
examples/basic/src/lib.rs: mutilating
examples/basic/src/lib.rs:4:5-4:12: `n += 1;` passed
examples/basic/src/lib.rs:14:9-14:16: `n += 1;` timed-out
examples/basic/src/lib.rs:21:5-21:12: `n += 1;` failed
examples/basic/src/lib.rs:28:18-28:27: `.join("")` nonbuildable
